provider/pkg/provider/invoke_helm_template.go

```diff
@@ -225,8 +228,14 @@ func (c *chart) template() (string, error) {
 	manifests := strings.Builder{}
 	manifests.WriteString(rel.Manifest)
 	for _, hook := range rel.Hooks {
-		manifests.WriteString("\n---\n")
-		manifests.WriteString(hook.Manifest)
+		switch {
+		case testAnnotation.MatchString(hook.Manifest):
+			// TODO: allow user to opt out via flag
```

Comment:
Should we log something here?

Reply:
Yeah, probably a good idea to log at v9.